### DISCUSSION TITLE
Sizers - GridSplitter not available for .Net 8?

### DISCUSSION THREAD

**Discussion by datovy-edam** [unoplatform/uno#18151] [Q&A] [answered]
I have an Uno Platform project produced using VS (latest) Uno Templates and it works as expected.  I added a grid and 2 columns and want to add a splitter in between.  At that point I tried adding "CommunityToolkit.WinUI.UI.Controls" (CTWUC) and is marked as deprecated, also I tried adding "Uno.CommunityToolkit.WinUI.UI.Controls" with the same results, it not made visible while stating the "using"...

It is stated (while adding CTWUC) that the library is deprecated but instead the "CommunityToolkit.WinUI.Controls.Sizers" (8.1.240821 latest stable version) should be used instead.  That I tried but still is not made visible either.

Can you recommend a solution to this?

**Comment by MartinZikmund** [comment]
Indeed, the `CommunityToolkit.WinUI.Controls.Sizers` is the correct one in this case and should work correctly. Can you please create an issue with a simple repro that does not work? We will investigate it.

**Comment by datovy-edam** [comment]
Greatly appreciated the quick response!   I did just that and conclude that is too simple, once you create a new project just replace the MainPage with a grid (3 columns) and try to put the GridSplitter in the middle column.  I did found that the current version (8.1.240821) you will not be able to find the expected splitters in the namespace, but as soon as you change to the previous version (8.0.240109) you will be able to see the expected splitters...   Can you please take a look on why the "current version" is not surfacing the Sizer controls?

**Reply by MartinZikmund** [reply, marked as answer]
I have created an issue https://github.com/unoplatform/uno/issues/18215 to track this regression
